src/components/DownloadComponents/DownloadCardGroup/DownloadCardGroup.tsx

```diff
@@ -1,92 +1,78 @@
-import React, { ChangeEvent, FC, useEffect, useState } from "react";
+import React, { ChangeEvent, FC } from "react";
 import { Control, Controller } from "react-hook-form";
+import styled from "styled-components";
 
 import type {
   DownloadResourceType,
   DownloadFormProps,
 } from "../downloads.types";
-import DownloadCard from "../DownloadCard";
+import ResourceCard from "../ResourceCard";
 
 import { LessonDownloadsData } from "@/node-lib/curriculum-api";
-import { GridArea } from "@/components/Grid";
-import Flex from "@/components/Flex";
-import { Heading } from "@/components/Typography";
-import FieldError from "@/components/FormFields/FieldError";
-import Checkbox from "@/components/Checkbox";
 import Box from "@/components/Box";
+import { getBreakpoint } from "@/styles/utils/responsive";
 
-type DownloadCardGroupProps = {
+export type DownloadCardGroupProps = {
   downloads?: LessonDownloadsData["downloads"];
   control: Control<DownloadFormProps>;
   hasError?: boolean;
-  errorMessage?: string;
-  onSelectAllClick: () => void;
-  onDeselectAllClick: () => void;
-  preselectAll: boolean;
   triggerForm: () => void;
 };
 
+const DownloadCardGrid = styled(Box)`
```

Comment:
Could this be styled(Grid) - or maybe could the gird-template-areas functionally be added it the Grid component ? - Maybe it's a bit too custom for a generic grip component though

Reply:
good shout! I updated the Grid component to accept templates and used the Grid component directly